tools/cassandra/handler.go

```diff
@@ -150,11 +149,28 @@ func createKeyspace(cli *cli.Context) error {
 	}
 	err = doCreateKeyspace(*config, keyspace)
 	if err != nil {
-		return handleErr(fmt.Errorf("error creating Keyspace:%v", err))
+		return handleErr(fmt.Errorf("error creating Keyspace:%+v", err))
 	}
 	return nil
 }
 
+func checkHealth(cli *cli.Context) error {
```

Comment:
discussed directly to change the semantics to ensure people can expect an error in case of "bad health", ex. `validate..`